src/signals/IncidentMap/components/IncidentMap/utils.ts

```diff
@@ -0,0 +1,17 @@
+import type { Map } from 'leaflet'
+
+import { DEFAULT_ZOOM } from '../utils'
+
+/**
+ * To get a good user experience on mobile when using flyTo you want to zoom out to the default zoom when
+ * the zoom exceeds the default zoom, and zoom to a set MIN_ZOOM when zoom it too small.
+ */
+export const getZoom = (map: Map) => {
+  const currentZoom = map.getZoom()
+  const MIN_ZOOM = 12
+
+  const zoomOut = currentZoom > DEFAULT_ZOOM
+  const zoomIn = currentZoom < MIN_ZOOM
```

Comment:
a bit confusing. To me zoomOut should be maxZoomLevel and zoomIn minZoomLevel. Then its semantically a little clearer

Reply:
Renamed a little.